python/res/job_queue/job_queue_manager.py

```diff
@@ -58,80 +55,62 @@ def __init__(self, queue):
         self.queue = queue
         super(JobQueueManager, self).__init__(c_ptr)
 
-    def status_timestamp(self):
-        """
-        Will return a datetime object corresponding to the last status change
-        """
-        ts = self._status_timestamp()
-        return ts.datetime()
-
-    def progress_timestamp(self, job_index = None):
-        """Will return the timestamp of last progress update.
-
-        By default the timestamp will be global, i.e. for all jobs, but if you
-        pass a job_index as optional argument the timestamp will correspond to
-        that job.
-        """
-        if job_index is None:
-            ts = self._global_progress_timestamp()
-        else:
-            ts = self._iget_progress_timestamp(job_index)
-
-        return ts.datetime()
-
-
     def get_job_queue(self):
```

Comment:
I suggest that we in the future we rewrite this so that `queue` is a private field and then expose a property named `queue` instead of the `get_job_queue`. We could already now, to be compliant with this, use `self.queue` instead of `self.get_job_queue()` internally in job_queue_manager.

Reply:
agree, it's fixed now:
```python
@property
    def queue(self):
        return self._queue
```